contrib/forms/review_of_systems/templates/review_of_systems/general_new.html

```diff
@@ -4,7 +4,7 @@
 
 {literal}
 
- <style type="text/css" title="mystyles" media="all">
+ <style title="mystyles" media="all">
```

Comment:
This will probably later on need to be revised -- title attributes here are unnecessary and the media attribute is dead in favor of doing it inside the stylesheet